aspnetcore/log-mon/metrics/metrics/samples/web-metrics/WebMetric.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.5.0-rc.1" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.5.1" />
+    <PackageReference Include="OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.6.0-rc.1" />
```

Comment:
I had to update these because I received the following error:

```
 Unhandled exception. System.MethodAccessException: Attempt by method 'Microsoft.Extensions.DependencyInjection.OpenTelemetryServicesExtensions.AddOpenTelemetry(Microsoft.Extensions.DependencyInjection.IServiceCollection)' to access method 'OpenTelemetry.Internal.Guard.ThrowIfNull(System.Object, System.String)' failed.> at Microsoft.Extensions.DependencyInjection.OpenTelemetryServicesExtensions.AddOpenTelemetry(IServiceCollection services)
 at Program.<Main>$(String[] args) in C:\Users\timde\dev\sandbox\WebMetric\Program.cs:line 4
```